src/Components/Ticket/Ticket.tsx

```diff
@@ -0,0 +1,277 @@
+import styled from "@emotion/styled";
+import Atropos from "atropos/react";
+
+import { jsconfTheme, ViewportSizes } from "../../../styles/theme";
+
+import { UserType, OwnTicket } from "../../helpers/API/types";
+
+const StyledAtropos = styled(Atropos)`
+  width: 350px;
+  height: 480px;
+  margin: 0 auto 16px;
+
+  @media (min-width: ${ViewportSizes.TabletLandscape}px) {
+    width: 680px;
+    height: 380px;
+  }
+
+  .atropos {
+    position: relative;
+    display: block;
+    perspective: 1200px;
+    transform: translate3d(0, 0, 0);
+  }
+  .atropos-rotate-scroll-x,
+  .atropos-rotate-scroll-y,
+  .atropos-rotate-touch {
+    -webkit-tap-highlight-color: transparent;
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+  }
+  .atropos-rotate-touch-scroll-y {
+    touch-action: pan-y;
+  }
+  .atropos-rotate-touch-scroll-x {
+    touch-action: pan-x;
+  }
+  .atropos-rotate-touch {
+    touch-action: none;
+  }
+  .atropos-rotate,
+  .atropos-scale {
+    width: 100%;
+    height: 100%;
+    transform-style: preserve-3d;
+    transition-property: transform;
+    display: block;
+  }
+  .atropos-highlight,
+  .atropos-shadow {
+    position: absolute;
+    pointer-events: none;
+    transition-property: transform, opacity;
+    display: block;
+    opacity: 0;
+  }
+  .atropos-shadow {
+    z-index: -1;
+    background: #000;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    filter: blur(30px);
+  }
+  .atropos-highlight {
+    left: -50%;
+    top: -50%;
+    width: 200%;
+    height: 200%;
+    background-image: radial-gradient(
+      circle at 50%,
+      rgba(255, 255, 255, 0.25),
+      transparent 50%
+    );
+    z-index: 0;
+  }
+  .atropos-rotate {
+    position: relative;
+  }
+  .atropos-inner {
+    width: 100%;
+    height: 100%;
+    position: relative;
+    overflow: hidden;
```

Comment:
Si le removemos esto, el wrapper no hace desaparecer el contenido (En caso q agreguemos más `data-atropos-offset`)

Estuve jugando con atropos, y se ve la raja weon <3 q onda lo cool 😆 

Cache algunos de los bugs q mencionabas, con mobile. por lo q estaba cachando, hay un tema con los paddings/margins y widths. 

Por lo que estaba mirando en el home de su pagina https://atroposjs.com/, los ejemplos son super planos/no-nesteados 🤔 

Se basan caleeeta en el position absolute, y pa hacer un efecto de profundida en las imágenes (Que no se corten), usan widths/heights mayores a 100%, y position en % negativos 🤔 

![image](https://user-images.githubusercontent.com/952992/204436886-6e48eb54-8967-4d80-a359-820b79bcb29d.png)

Reply:
e.g checa

https://user-images.githubusercontent.com/952992/204437407-d56e2e5c-0f49-488f-89df-e60460aae7c7.mp4

Reply:
me retracto con lo del overflow del comienzo, si tienes imágenes q sobresalgan, y no hay un `overflow: hidden` entonces el efecto se activa de acuerdo al hover en la imágen (aunque no se ve), y no de acuerdo al hover en el elemento atropos

Reply:
JIC it helps https://github.com/JSConfCL/2022/pull/104